docs/configuration.rst

```diff
@@ -0,0 +1,141 @@
+.. _configuration:
+
+Configuration Specifications
+============================
+This page documents the various options and
+parameters that can be set in the configuration
+file.
+
+MetSim Section
+--------------
+
+**Required Variables**
+
+``time_step :: int``: The timestep to disaggregate in minutes.  If given as 1440
```

Comment:
MetSim should only be used with daily input data.  The code *might* work with subdaily data - but the results will be messed up.